app/views/records/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render 'shared/navbar', back_link: group_records_path, title: "New Transaction" %>
+
+<%= form_with model: [@group, @record], class:"add-form" do |form| %>
+  <h1 class="form_title">Add a new Transaction</h1>
+
+<div class="form-fields">
+  <div class="field">
+    <%= form.label :name, class: "form-label" %>
+    <%= form.text_field :name, class: "form-control" %>
+  </div>
+
+  <div class="field">
+    <%= form.label :amount, class: "form-label" %>
+    <%= form.number_field :amount, class: "form-control" %>
+  </div>
+
+  <div class="field">
+    <%= form.label :group_ids, class: "form-label" %>
+    <%= form.collection_select :group_ids, Group.all, :id, :name, {}, class: "form-select" %>
+  </div>
+  </div>
+
+  <%= form.submit "Save", class: "submit-btn" %>
+<% end %>
```

Comment:
- [ ] as part of the requirements kindly add some validations errors message and some success/fail paths where the user can see the error so he can fix it  , kindly fix it for all of your inputs & forms 🙏 


![sasssssssssssssssssssssssss](https://github.com/mnaibei/budget_app/assets/89282221/41f91826-e4d9-4036-8508-abcdac6a4b0e)